io/hdfs/src/test/java/org/apache/beam/io/hdfs/WritableCoderTest.java

```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package com.google.cloud.dataflow.contrib.hadoop;
+package org.apache.beam.io.hdfs;
```

Comment:
Is it related to the sdk ? For me (correct me if I'm wrong): sdk is the programming model, java-sdk is an impl of this model in java, scala-sdk will be an impl of the model in scala. As a developer, I can use hdfs io in java or scala sdk.